src/test/resources/reference.conf

```diff
@@ -55,6 +55,13 @@ googleServices {
       workerCount = 1
     }
   }
+
+  kms {
+    project = "broad-dsde-dev"
+    location = "global"
+    keyRingId = "mtalbott"
+    keyId = "mtalbott"
```

Comment:
shall we change `keyRingId` and `keyId` to something else that's not specific to you